### ISSUE TITLE
not able to login using the generated passwd on docker instance

### ISSUE THREAD

**Issue by jcran** [intrigueio/intrigue-core#45]
As reported in the Gitter chan: 

```
fyoorer @aditya_gujar_twitter 02:05
just set up intrigue docker in OSX , but not able to login using the generated passwd

Jonathan Cran @jcran 02:06
hmm, services may not have restarted with the new config
one sec
try... AUTOGENERATED
all caps

fyoorer @aditya_gujar_twitter 02:20
i just did docker build . and then docker run

fyoorer @aditya_gujar_twitter 02:21
here you go:
Starting PostgreSQL 9.6 database server G G[ OK ]
Starting redis-server: redis-server.
Starting intrigue-core processes
[+] Setup initiated!
[+] Generating system password: cnznqhgpoqlidrts
[+] Copying puma config....
[ ] File already exists, skipping: /core/config/puma.rb
[+] Copying system config....
[ ] File already exists, skipping: /core/config/config.json
[+] Copying database config....
[+] Creating.... /core/config/database.yml
[+] Setting up task worker config....
[+] Copying: /core/config/sidekiq-task-interactive.yml.default
[+] Copying: /core/config/sidekiq-task-autoscheduled.yml.default
[+] Copying: /core/config/sidekiq-app.yml.default
[+] Obtaining latest data...
Getting data files (will fail if we don't have internet)
Getting latest MaxMind GeoLiteCity database
Getting latest web_accounts_list.json
[+] Copying control script...
[ ] File already exists, skipping: /core/util/control.sh
[+] Schema Version: 5
Starting interactive task processing...
Starting autoscheduled task processing...
Starting background worker processing...
Starting puma...
Setting puma environment to: development
Puma starting in single mode...
Version 3.6.2 (ruby 2.3.1-p112), codename: Sleepy Sunday Serenity
Min threads: 0, max threads: 16
Environment: development
Listening on tcp://0.0.0.0:7777
Use Ctrl-C to stop

Jonathan Cran @jcran 02:22
perfect
for some reason it thought your /core/config/config.json already existed
so it didn't overwrite with the new pass

fyoorer @aditya_gujar_twitter 02:25
i am checking in docker shell, core/config/puma.rb is not there

Jonathan Cran @jcran 02:25
can you ls your /core/config?

fyoorer @aditya_gujar_twitter 02:27
sorry. i was checking somewhere else
config.json database.yml puma.rb sidekiq-app.yml sidekiq-task-autoscheduled.yml sidekiq-task-interactive.yml config.json.default database.yml.default puma.rb.default sidekiq-app.yml.default sidekiq-task-autoscheduled.yml.default sidekiq-task-interactive.yml.default

Jonathan Cran @jcran 02:28
kk, and can you do the same in the repo (on the docker host, not inside the container)
ls /path/to/intrigue-core/config

fyoorer @aditya_gujar_twitter 02:29
config.json database.yml.default sidekiq-app.yml.default sidekiq-task-interactive.yml.default config.json.default puma.rb.default sidekiq-task-autoscheduled.yml.default
```

**Comment by jcran** [comment]
Current thinking is that this is a config issue, as the core repo is copied over to the Docker host, then a rake db:migrate is run, which copies all the .default files to their respective locations and updates the system password. In this case, the config.json existed on the host - so it wasn't updated with the password.

**Comment by jcran** [comment]
Updated documentation in the README which instructed users to create a config.json and configure it locally in the repo before starting docker. This was messing with the password auto-generation. (The config.json with the new password was not being created because the file already existed).

**Comment by jcran** [comment]
Fixed in v0.3
